packages/vitest/src/node/index.ts

```diff
@@ -248,8 +250,17 @@ class Vitest {
   }
 
   async close() {
```

Comment:
async without await here

Reply:
it is not required that async functions contain an await and close returns a promise, so the contract of async is there.
And while it is not strictly needed here i think it doesn't hurt and shows the async nature of the implementation.